tests/python/unittest/test_runtime_graph_debug.py

```diff
@@ -19,26 +19,34 @@
 import re
 import sys
 import time
+from distutils.log import debug
 
+import numpy as np
 import pytest
-
 import tvm
 import tvm.testing
-from tvm import te
-import numpy as np
-from tvm import rpc
+from tvm import rpc, te
+from tvm._ffi.base import TVMError
 from tvm.contrib import utils
 from tvm.contrib.debugger import debug_executor
 
+# Constants for creating simple graphs
+n = 4
+A = te.placeholder((n,), name="A")
+B = te.compute(A.shape, lambda *i: A(*i) + 1.0, name="B")
+s = te.create_schedule(B.op)
```

Comment:
Put these inside the functions using them or use fixtures. Freestanding globals like this may cause issues with multiprocessing.

Reply:
Done